server/channels/store/searchtest/helper.go

```diff
@@ -202,12 +196,7 @@ func (th *SearchTestHelper) deleteBotUser(botID string) error {
 	return th.Store.User().PermanentDelete(th.Context, botID)
 }
 
-func (th *SearchTestHelper) cleanAllUsers() error {
-	users, err := th.Store.User().GetAll()
```

Comment:
This was needed because it was deleting all users in the DB, and not just the ones created during the test, causing further failures in future tests.